.ci/test-install-docs.sh

```diff
@@ -12,9 +12,39 @@ info()
 	echo "INFO: $msg"
 }
 
-# Detect if any installation documents changed. If so, run the
-# kata manager to "execute" the install guide to ensure the
-# commands it specified result in a working system.
+# Run the kata manager to "execute" the install guide to ensure the commands
+# it specified result in a working system.
+test_distro_install_guide()
+{
+	local -r GOPATH=$(go env GOPATH)
+	[ -z "$GOPATH" ] && die "cannot determine GOPATH"
+
+	local -r mgr="${GOPATH}/src/github.com/kata-containers/tests/cmd/kata-manager/kata-manager.sh"
+
+	[ ! -e "$GOPATH" ] && die "cannot find $mgr"
+
+	source /etc/os-release
```

Comment:
Expand to cover the multiple potential source files as discussed on https://github.com/kata-containers/tests/issues/589, maybe like:

```
source /etc/os-release || source /usr/lib/os-release
```